src/status_im/ui2/screens/chat/composer/images/view.cljs

```diff
@@ -3,11 +3,13 @@
             [quo2.foundations.colors :as colors]
             [re-frame.core :as rf]
             [react-native.core :as rn]
-            [status-im.ui2.screens.chat.composer.style :as style]))
+            [status-im.ui2.screens.chat.composer.style :as style]
+            [status-im.utils.platform :as platform]))
 
 (defn image
   [item]
-  [rn/view
+  [rn/view (when platform/android?
+             {:style {:margin-top 8}})
```

Comment:
this looks like magic, why we need to add margin-top and only for Android, looks not obvious

Reply:
but i mean it shouldn't be done in every view, this padding should set only once somewhere in general view i guess